CMakeLists.txt

```
@@ -50,10 +50,12 @@ set_property(CACHE ACE_LIBRARY_KIND PROPERTY STRINGS ${ACE_LIBRARY_KINDS})
 set(ACE_DEBUG OFF CACHE BOOL "Build with ACE-specific debug/safety functionality.")
 set(ACE_DEBUG_UAE OFF CACHE BOOL "With ACE_DEBUG enabled, output log to UAE console.")
 set(ACE_BOB_WRAP_Y ON CACHE BOOL "Conrols Y-wrapping support in bob manager. Disable for extra performance in simple buffer scenarios.")
+set(ACE_USE_ECS_FEATURES OFF CACHE BOOL "Enable ECS only feature sets.")
```

This would also work for AGA, so affects everything ECS and up

```suggestion
set(ACE_USE_ECS_FEATURES OFF CACHE BOOL "Enable ECS feature sets.")
```

On a side note - perhaps "allow" is better word than "USE"/"enable"?